### PR TITLE
VZ-4626 Cleanup VMO logging

### PR DESCRIPTION
**Pull request by pfmackin**
This is a first pass at cleaning up VMO logging.

### CODE REVIEW

File: pkg/util/logs/loginit.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2020, 2022, Oracle and/or its affiliates.
```
Comment (kminder) [inline pkg/util/logs/loginit.go:1]: Just 2022 right?

Comment (pfmackin) [reply pkg/util/logs/loginit.go]: fixed

File: pkg/resources/statefulsets/statefulset_test.go [hunk 2]
```diff
@@ -5,6 +5,7 @@ package statefulsets
 
 import (
 	"fmt"
+	"github.com/verrazzano/verrazzano-monitoring-operator/pkg/util/logs/vzlog"
```
Comment (kminder) [inline pkg/resources/statefulsets/statefulset_test.go:8]: Missed this copyright.

File: pkg/vmo/ingress.go [hunk 3]
```diff
@@ -31,7 +31,7 @@ func CreateIngresses(controller *Controller, vmo *vmcontrollerv1.VerrazzanoMonit
 		vmo.Spec.IngressTargetDNSName = controller.operatorConfig.DefaultIngressTargetDNSName
 	}
 	var ingressNames []string
-	zap.S().Infof("Creating/updating Ingresses for vmo '%s' in namespace '%s'", vmo.Name, vmo.Namespace)
+	controller.log.Oncef("Creating/updating Ingresses for vmo '%s' in namespace '%s'", vmo.Name, vmo.Namespace)
```
Comment (kminder) [inline pkg/vmo/ingress.go:34]: Not sure about the use of vmo here.  Should it be capitalized or spelled out?

File: pkg/vmo/ingress.go [hunk 4]
```diff
@@ -65,15 +65,15 @@ func CreateIngresses(controller *Controller, vmo *vmcontrollerv1.VerrazzanoMonit
 	}
 
 	// Delete ingresses that shouldn't exist
-	zap.S().Infof("Deleting unwanted Ingresses for vmo '%s' in namespace '%s'", vmo.Name, vmo.Namespace)
+	controller.log.Oncef("Deleting unwanted Ingresses for vmo '%s' in namespace '%s'", vmo.Name, vmo.Namespace)
```
Comment (kminder) [inline pkg/vmo/ingress.go:68]: vmo?

File: pkg/vmo/ingress.go (hunk 3 above)
Comment (pfmackin) [reply pkg/vmo/ingress.go:34]: I think abbreviation is fine, though it probably should be VMI.

File: pkg/resources/statefulsets/statefulset_test.go (hunk 2 above)
Comment (pfmackin) [reply pkg/resources/statefulsets/statefulset_test.go:8]: fixed

File: pkg/vmo/ingress.go (hunk 3 above)
Comment (pfmackin) [reply pkg/vmo/ingress.go]: fixed

File: pkg/vmo/pvc.go [hunk 5]
```diff
@@ -39,7 +39,7 @@ func createPersistentVolumeClaims(controller *Controller, vmo *vmcontrollerv1.Ve
 	}
 	deploymentToAdMap := map[string]string{}
 
-	zap.S().Infof("Creating/updating PVCs for vmo '%s' in namespace '%s'", vmo.Name, vmo.Namespace)
+	controller.log.Oncef("Creating/updating PVCs for vmo '%s' in namespace '%s'", vmo.Name, vmo.Namespace)
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply pkg/vmo/pvc.go]: fixed

File: pkg/util/logs/loginit.go [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+	} else {
+		config = zap.NewProductionConfig()
+	}
+	if opts.Level != nil {
+		config.Level = opts.Level.(zap.AtomicLevel)
+	} else {
+		config.Level.SetLevel(zapcore.InfoLevel)
+	}
+	config.EncoderConfig.EncodeTime = zapcore.TimeEncoderOfLayout(timeFormat)
+	config.EncoderConfig.TimeKey = "@timestamp"
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply pkg/util/logs/loginit.go:29]: fixed

**Review by kminder — APPROVED** [review]
Approved

**Pull request merged**